README.yaml

```diff
@@ -0,0 +1,106 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: ansible-ntp
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "MIT"
+
+copyrights:
+  - name: "Jeff Geerling"
+    url: "https://github.com/geerlingguy"
+    year: "2017"
+
+# Canonical GitHub repo
+github_repo: cloudposse/ansible-ntp
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/ansible-ntp.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/ansible-ntp"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/ansible-ntp.svg"
```

Comment:
This is not working depsite having releases. Is there a typo some where?

Reply:
Not, because of there is no one release, just tags.
Fixed due publish new release.
[source](https://github.community/t5/How-to-use-Git-and-GitHub/Why-last-release-is-not-marked-as-quot-latest-release-quot/td-p/1762)